boards/p-nucleo-wb55/doc.txt

```diff
@@ -0,0 +1,109 @@
+/**
+@defgroup    boards_p-nucleo-wb55 STM32 p-nucleo-wb55
+@ingroup     boards_common_nucleo
+@brief       Support for the STM32 p-nucleo-wb55
+
+Hardware
+================
+
+![st-nucleo-wb55](https://miro.medium.com/max/700/1*9OG-4Ix4EzHX9uBpMve2IA.jpeg)
+
+MCU
+-------------------
+
+| MCU        | STM32WB55RG                     |
+|:---------- |:------------------------------- |
+| Family     | ARM Cortex-M4                   |
+| Vendor     | ST Microelectronics             |
+| RAM        | 256Kb                           |
+| Flash      | 512Kb                           |
+| Frequency  | 64MHz                           |
+| FPU        | yes                             |
+| Timers     | 8 (3x 16-bit, 1x 32-bit [TIM5]) |
+| LPTimers   | 2x 16-bit                       |
+| ADCs       | 1x 19-channel 12-bit            |
+| UARTs      | 1                               |
+| LUARTs     | 1                               |
+| SPIs       | 1                               |
+| I2Cs       | 2                               |
+| RTC        | 1                               |
+| Vcc        | 1.65V - 3.6V                    |
+| Datasheet  | [Datasheet](https://www.st.com/resource/en/datasheet/stm32wb55cc.pdf)|
+| Reference Manual | [Reference Manual](https://www.st.com/resource/en/datasheet/stm32wb55cc.pdf) |
+| User Manual | [User Manual](https://www.st.com/content/ccc/resource/technical/document/user_manual/group1/13/58/22/1a/f2/ff/43/5c/DM00517423/files/DM00517423.pdf/jcr:content/translations/en.DM00517423.pdf) |
+
+Overview
+================
+
+The p-nucleo-wb55 is a  is a multi-protocol wireless and ultra-low-power
+device embedding a powerful and ultra-low-power radio compliant with the
+Bluetooth® Low Energy (BLE) SIG specification v5.0 and with IEEE
+802.15.4-2011.board It integrates an ARM Cortex-M4 STM32WB55RG microcontroller
+with 256Kb of RAM and 1Mb of ROM Flash.
+
+Flashing the device
+-------------------
+
+The ST p-nucleo-wb55 board includes an on-board ST-LINK programmer and can be
```

Comment:
Needs to be updated with instructions on how to setup pyocd for this board, see https://github.com/RIOT-OS/RIOT/pull/11895#issuecomment-573685539